routers/api/v1/repo/issue_test.go

```diff
@@ -0,0 +1,73 @@
+package repo_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/go-gitea/gitea/models"
+	"github.com/go-gitea/gitea/testutil"
+	api "github.com/gogits/go-gogs-client"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestIssuesAPI(t *testing.T) {
+	Convey("Given the issue API", t, func() {
+		testutil.TestGlobalInit()
+
+		Convey("It should return issues from the index", func() {
+			testutil.PrepareTestDatabase()
+
+			w, r := testutil.NewTestContext("GET", "/api/v1/repos/user1/foo/issues", "", nil, "1")
+			testutil.ServeHTTP(w, r)
+			So(w.Code, ShouldEqual, http.StatusOK)
```

Comment:
I see, but it useless info. `/api/v1/repos/user1/foo/issues` may returns 200 and `{"foo": "bar"}` in response body. Or `{"oops": "something went wrong"}`. Or `Fuck the json`, etc…